examples/Ethereum Crypto Trend Analysis Notebook.ipynb

```diff
@@ -0,0 +1,487 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Ethereum Cryopto Trend Analysis with OpenBB\n",
+        "\n",
+        "## Description\n",
+        "This notebook showcases the application of technical analysis techniques to explore Ethereum price trends and volatility. It utilizes OpenBB's historical data to calculate and visualize moving averages, analyze trading volume, and assess price volatility. The notebook demonstrates how these tools can be employed to gain insights into market dynamics and potentially inform investment decisions.\n",
+        "\n",
+        "#### Author\n",
+        "[MacBobby Chibuzor](https://github.com/theghostmac)\n",
+        "\n",
+        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1KwDtij9ln1UjdJOYKOGltUkaPctrqWGz?authuser=0#scrollTo=HeB3TlvkmFoK)\n",
+        "\n",
+        "The dependencies for running this includes openbb, pandas, and matplotlib.\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "HeB3TlvkmFoK"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install openbb"
```

Comment:
If you run `!pip install openbb -q`  -- that will suppress the installs and make it look nicer

Reply:
I have updated the first cell to use this instead.